packages/vkui/src/components/Cell/Cell.test.tsx

```diff
@@ -132,4 +134,83 @@ describe('Cell', () => {
       expect(clickStub).toHaveBeenCalledTimes(1);
     });
   });
+
+  it('check selectable mode', () => {
+    render(
+      <Cell mode="selectable" data-testid="cell">
+        Саша Колобов
+      </Cell>,
+    );
+    const cell = screen.getByTestId('cell');
+    expect(cell.tagName).toBe('LABEL');
+    expect(cell.parentElement).toHaveClass(styles['Cell--selectable']);
+
+    const checkbox = getByRole(cell, 'checkbox');
+    expect(checkbox).toBeInTheDocument();
+  });
+
+  it('check dragging className add when dragging cell', async () => {
+    render(
+      <Cell data-testid="list-xyz" draggable draggerLabel={label}>
+        xyz
+      </Cell>,
+    );
+
+    const cell = screen.getByTestId('list-xyz');
+    const dragger = cell.querySelector(`.${styles['Cell__dragger']}`) as HTMLElement;
+    expect(dragger).toBeInTheDocument();
+
+    fireEvent.mouseDown(dragger, {
+      clientX: 0,
+      clientY: 0,
+    });
+    fireEvent.mouseMove(dragger, {
+      clientX: 0,
+      clientY: 100,
+    });
+
+    expect(cell.parentElement).toHaveClass(styles['Cell--dragging']);
+  });
+
+  it('check that dragger after the content in IOS', () => {
+    render(
+      <ConfigProvider platform={Platform.IOS}>
+        <Cell data-testid="list-xyz" draggable draggerLabel={label}>
+          <div data-testid="content"></div>
+        </Cell>
+      </ConfigProvider>,
+    );
+    const cell = screen.getByTestId('list-xyz');
+    const dragger = cell.querySelector(`.${styles['Cell__dragger']}`) as HTMLElement;
+    const content = screen.getByTestId('content');
+
+    // Проверяем, что dragger находится в DOM дереве после контента, значит в after
+    expect(
+      content.compareDocumentPosition(dragger) & Node.DOCUMENT_POSITION_FOLLOWING,
+    ).toBeTruthy();
+  });
+
+  it('should have disabled classNames', () => {
+    render(
+      <Cell data-testid="list-xyz" draggable draggerLabel={label} disabled>
+        xyz
+      </Cell>,
+    );
+
+    const cell = screen.getByTestId('list-xyz');
+    const dragger = cell.querySelector(`.${styles['Cell__dragger']}`) as HTMLElement;
```

Comment:
```suggestion
    const dragger = cell.querySelector<HTMLElement>(`.${styles['Cell__dragger']}`)!;
```
чтобы без `as`

PS: единственное, `!` надо добавить, чтобы на `null` не ругался 🌚

Reply:
ну и также в других местах в тесте

Reply:
Поправил